### ISSUE TITLE
Feature request: Gas (or battery) mileage calculation

### ISSUE THREAD

**Issue by the-static** [khpylon/MachEWidget#42]
Not an issue, but a feature request. I don't know how easy this would be to implement, but it would be neat if there was a fuel mileage calculation. I'd think if it could somehow look at mileage whenever the fuel level hits 100 and how many miles elapsed since the last time as well as how many gallons it estimates based on the percentage at fill up? Obviously this is easy to do manually, but having a way to keep track of it automatically, separate from the trucks own calculation could be cool.

**Comment by khpylon** [comment]
I'll have to think about the feasibility.  The app only gets new data when it's uploaded to the servers by the vehicle, so it might be difficult to calculate anything reasonable.   I'm recommending against forced updates with the current API since it could result in the account being locked.   Perhaps when Ford publishes their official public API (which could be soon) this might be possible.

**Comment by the-static** [comment]
Fair enough, just an idea I thought might be interesting to try, no worries.
